main/dns-root-hints/APKBUILD

```diff
@@ -13,27 +13,34 @@ options="net"
 source="verisign-grs-nstld-key.asc named.root named.root.sig"
 builddir="$srcdir/$pkgname-$pkgver"
 
-build() {
-	mkdir -p "$builddir"
+_check_sig() {
+	local GNUPGHOME="$builddir/.gpg"
+	install -d -m 0700 "$GNUPGHOME"
+	gpg --import < "$srcdir"/verisign-grs-nstld-key.asc
+	gpg --verify "${_tmp}"/named.root.sig "${_tmp}"/named.root
+	rm -rf "$GNUPGHOME"
+	mv "${_tmp}"/named.root.sig "${_tmp}"/named.root "$builddir"/
```

Comment:
I'd move the files in build() because it's not strictly part of the _check_sig() process

Reply:
It absolutely is. Unverified files should not be moved.

Reply:
If _check_sig() fails, it won't move on into build().

Reply:
Indeed. You are arguing for moving the operation of moving verified files out of the `_check_sig` function to every place that function is called. Why does that seem better to you?

Reply:
By saying "I'd" (I would) usually I mean a preference, not a strong opinion and that's the general meaning in english I believe but I could be mistaken since I'm not a native english speaker.

For the sake of the argument in this case "every place" equals to 1.

If you have a strong opinion on this you may leave it where it is. :smile: